front/app/containers/IdeasIndexPage/index.tsx

```diff
@@ -111,11 +114,12 @@ export default () => {
                 when fitlers are changed in the IdeaCardsWithFiltersSidebar component and scrollToTopIdeasList util.
             */}
             <IdeaListScrollAnchor />
-            <IdeaCardsWithFiltersSidebar
-              invisibleTitleMessage={messages.a11y_IdeasListTitle1}
-              ideaQueryParameters={ideasQueryParameters}
-              onUpdateQuery={updateSearchParams}
-            />
+            <Suspense fallback={<Spinner />}>
+              <IdeasWithFiltersSidebar
```

Comment:
I would not have lazy-loaded here is what I meant because this will load super fast since it's visible on load (while it's not on the project page), but it doesn't matter that much (page is not important) so approving! But feel free to still remove it.